src/bgg_companion_api.py

```diff
@@ -12,91 +12,103 @@
 from cachetools import cached, TTLCache
 
 
-def request(url: str):
-    response = RequestsRetryClient().request(method="GET", url=url)
-    return response.text
+class BggCompanionApi(object):
```

Comment:
Just a reminder: did you update the `app.py` file to work with the new format for this module? Or is that abstracted away somehow?

Reply:
Ahhhh yes yes, this was one of the many changes I had locally all messed up and not committed to the branch